Sources/StreamChatUI/UIConfig+SwiftUI.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+@dynamicMemberLookup
+/// `UIConfig` represented as `ObservableObject` class for SwiftUI requirements.
+public class SwiftUIConfig: ObservableObject {
```

Comment:
`UIConfig` is generic over extra data. Let's follow the same naming pattern we use elsewhere:
```swift
public typealias SwiftUIConfig = _SwiftUIConfig<NoExtraData>
public class _SwiftUIConfig<ExtraData: ExtraDataTypes> {}
```

Maybe instead of a top-level type, we could namespace the wrapper?
```swift
extension UIConfig {
    class ObservableObject: SwiftUI.ObservableObject {}
}
```

Reply:
Adjusted in d41dac45bd4ebf0c747a410ded539c4210bba194.